deltachat-ios/View/ChatTitleView.swift

```diff
@@ -49,10 +38,11 @@ class ChatTitleView: UIView {
         let containerView = UIView()
         containerView.translatesAutoresizingMaskIntoConstraints = false
         addSubview(containerView)
+
         addConstraints([ containerView.constraintAlignTopTo(self),
                          containerView.constraintAlignBottomTo(self),
                          containerView.constraintCenterXTo(self),
-                         containerView.constraintWidthTo(UIScreen.main.bounds.width - CGFloat(paddingNaviationButtons + sizeStreamingIndicator)) ])
```

Comment:
I think the removal of this line introduces the following bug:
![Screen Shot 2020-05-27 at 12 27 18](https://user-images.githubusercontent.com/2614900/83009156-92bff600-a016-11ea-92ce-d69963afc23a.png)

Reply:
Turns out this ticket produces more bugs than it actually fixes.

Reply:
Thanks for finding this bug. I fixed it.